app/views/addresses/edit.html.haml

```diff
@@ -1,8 +1,9 @@
 %header.entry-header
   %h1.entry-title= t('.title', company_name: @company.name)
-  = t('.cannot_delete_address') if @address.mail
-  %span.glyphicon.glyphicon-info-sign{ title: "#{t('.delete_address_tooltip')}",
-                                       data: {toggle: 'tooltip'} }
+  - if @address.mail
+    = t('.cannot_delete_address') if @address.mail
```

Comment:
don't need the `if @address.mail` at the end of this line

Reply:
Oops .... that was sloppy of me.  Thanks.